community-build/src/scala/dotty/communitybuild/projects.scala

```diff
@@ -792,6 +805,7 @@ def allProjects = List(
   projects.scalacheckEffect,
   projects.fs2,
   projects.libretto,
+  projects.jacksonModuleScala,
```

Comment:
Why isn't the java8-compat included in this list?

Reply:
apologies - missed that, I've fixed up the PR to include that